HGustavs/LenaSYS
# Removed unwanted css to make the #NewTabLink element adhere in light mode

Testing:

1. Log in as stei
2. Go to course Webbprogrammering
3. Check that the icons is purple in light mode and white in dark mode.

![image](https://user-images.githubusercontent.com/81631818/169964427-58cfaf30-db79-42a1-823e-bbe3347beb83.png)
![image](https://user-images.githubusercontent.com/81631818/169964493-6e037cf4-52bc-4daa-996a-0ae00f188f58.png)
